test/helper/launcher/launcher.go

```diff
@@ -0,0 +1,257 @@
+package launcher
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"strings"
+)
+
+const (
+	ExpectedContext = "kind-kind"
+
+	LauncherTestInstall = "test-ako"
+
+	HelmRepoURL = "https://mongodb.github.io/helm-charts"
+
+	RepoRef = "mongodb"
+
+	OperatorChart = "mongodb-atlas-operator"
+
+	AtlasURI = "https://cloud-qa.mongodb.com"
+
+	// #nosec G101 -- This is just a name
+	AtlasSecretName = "mongodb-atlas-operator-api-key"
+)
+
+type Launcher struct {
+	credentials   AtlasCredentials
+	version       string
+	cmdOutput     bool
+	clearKind     bool
+	clearOperator bool
+	clearSecret   bool
+	appliedYAMLs  []string
+}
+
+// NewLauncher creates a new operator Launcher
+func NewLauncher(creds AtlasCredentials, version string, cmdOutput bool) *Launcher {
+	return &Launcher{credentials: creds, version: version, cmdOutput: cmdOutput, appliedYAMLs: []string{}}
+}
+
+// NewFromEnv creates an operator Launcher using defaults and credentials from env vars
+func NewFromEnv(version string) *Launcher {
+	return NewLauncher(credentialsFromEnv(), version, true)
+}
+
+// MustSetEnv sets the env var value given, or panics if the env var is not set
+func MustSetEnv(envvar string) string {
```

Comment:
the method name `MustSetEnv` is confusing. As per Go idioms `Set` always implies an imperative action being performed, i.e `MustSetThisThing` which will panic if setting `ThisThing` fails. I suggest to rename to:
```suggestion
func MustLookupEnv(envvar string) string {
```

Reply:
coming next